doc/releases.md

```diff
@@ -91,7 +96,7 @@ The _CHANGELOG.md_ entry should take the following form:
 * Include the full list of commits since the last release here
 ```
 
-### 5. Update _src/node_version.h_
+### 4. Update _src/node_version.h_
```

Comment:
I know this is a tiny thing, but we should probably make a script for it. I know I've definitely messed this up in subtle ways before.